FinniversKit/Sources/Fullscreen/FavoriteSoldView/FavoriteSoldView.swift

```diff
@@ -0,0 +1,265 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+public protocol FavoriteSoldViewModel {
+    var title: String? { get }
+    var imageUrl: String? { get }
+    var bodyText: String { get }
+    var ribbonTitle: String { get }
+    var similarAdsTitle: String { get }
+    var retryButtonTitle: String { get }
+    var noRecommendationsTitle: String { get }
+}
+
+public protocol FavoriteSoldViewDelegate: AnyObject {
+    func favoriteSoldViewDidTapSoldFavorite(_ favoriteSoldView: FavoriteSoldView)
+    func favoriteSoldViewDidTapRetryButton(_ favoriteSoldView: FavoriteSoldView)
+}
+
+public class FavoriteSoldView: UIView {
+    public var model: FavoriteSoldViewModel? {
+        didSet {
+            if let model = model {
+                self.configure(with: model)
+            }
+        }
+    }
```

Comment:
Do you need to keep a reference to this model? Maybe it is enough to just call `configure(with:)`

Reply:
You're right, changed it now! 👍